backend/src/main/java/com/ody/route/domain/DepartureTime.java

```diff
@@ -11,9 +11,9 @@ public class DepartureTime {
 
     private final LocalDateTime value;
 
-    public DepartureTime(RouteTime routeTime, Meeting meeting) {
+    public DepartureTime(Meeting meeting, long estimatedMinutes) {
         LocalDateTime meetingTime = LocalDateTime.of(meeting.getDate(), meeting.getTime());
```

Comment:
[제안]
```suggestion
        LocalDateTime meetingTime = meeting.getMeetingTime();
```

아마 Meeting 도메인에 getMeetingTime을 하면 바로 trim된 LocalDateTime을 가져올 수 있을 거에요!

Reply:
[반영] [36d201c](https://github.com/woowacourse-teams/2024-ody/pull/439/commits/36d201c1e51f5b5a791cde69320df11a1fe09a84)